test/fixtures.spec.js

```diff
@@ -92,7 +93,78 @@ describe('fixtures', () => {
               };
               break;
 
+            case 'remove-context-wrap-es6':
```

Comment:
I am not sure on which cases, I should test `contextMode` on. So I did it on a bunch of them. 
Could you be more explicit on how you want me to proceed.